src/data/seq/seq.lean

```diff
@@ -72,15 +72,13 @@ protected def mem (a : α) (s : seq α) := some a ∈ s.1
 instance : has_mem α (seq α) :=
 ⟨seq.mem⟩
 
-theorem le_stable (s : seq α) {m n} (h : m ≤ n) :
-  s.nth m = none → s.nth n = none :=
-by {cases s with f al, induction h with n h IH, exacts [id, λ h2, al (IH h2)]}
+theorem le_stable (s : seq α) {m n} (h : m ≤ n) : s.nth m = none → s.nth n = none :=
+by { cases s with f al, induction h with n h IH, exacts [id, λ h2, al (IH h2)] }
 
 /-- If a sequence terminated at position `n`, it also terminated at `m ≥ n `. -/
-lemma terminated_stable (s : seq α) {m n : ℕ} (m_le_n : m ≤ n)
-(terminated_at_m : s.terminated_at m) :
+lemma terminated_stable : ∀ (s : seq α) {m n : ℕ}, m ≤ n → s.terminated_at m →
   s.terminated_at n :=
```

Comment:
Would be tempted to line wrap this as
```suggestion
lemma terminated_stable :
  ∀ (s : seq α) {m n : ℕ}, m ≤ n → s.terminated_at m → s.terminated_at n :=
```
although personally I'd have left things before the colon.